src/main/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializer.java

```diff
@@ -16,6 +16,7 @@
 
     private final MonetaryAmountFormatFactory factory;
     private final FieldNames names;
+    private final boolean numberAsString;
```

Comment:
We could implement `ConfigFeature` ourselves, but I think what would be a better fit would be an enum:

```java
public enum Style {
    DECIMAL, STRING
}
```